go/roothash/api/api.go

```diff
@@ -48,19 +48,8 @@ var (
 	_ cbor.Unmarshaler = (*MergeDiscrepancyDetectedEvent)(nil)
 )
 
-// Info contains information about a root hash backend.
-type Info struct {
```

Comment:
Removed this since this was only used in a single place https://github.com/oasislabs/oasis-core/pull/2333/files#diff-2b8a5495d9729944400cef9dece417edR361

and we can just query genesis there. Can revert this if we prefer it that way. (but in that case we might also want to move `commonWorker.GenesisDoc.RootHash.Parameters.TransactionScheduler` into the `Info` struct for consistency?)